### PR TITLE
TDQ-14484 broadcast dictionary to cluster

### PR DESCRIPTION
**Pull request by sizhaoliu**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDQ-14484

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [x] No, and no need to (backend changes only)

### CODE REVIEW

File: dataprep-backend-service/src/main/java/org/talend/dataprep/configuration/Analyzers.java [hunk 1]
```diff
@@ -47,9 +48,10 @@ public StatisticsAdapter statisticsAdapter() {
     @Bean
     public AnalyzerService analyzerService() {
         LOGGER.info("Data Quality strategy is {} and located in {}", luceneIndexStrategy, dataqualityIndexesLocation);
-        return new AnalyzerService(dataqualityIndexesLocation, //
-                luceneIndexStrategy, //
-                CategoryRecognizerBuilder.newBuilder().lucene());
+        //return new AnalyzerService(dataqualityIndexesLocation, //
```
Comment (fhuaulme) [inline dataprep-backend-service/src/main/java/org/talend/dataprep/configuration/Analyzers.java]: No commented out code, please.

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1023/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1023/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1024/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1024/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1050/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1050/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1075/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1075/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1086/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1086/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1093/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/1093/)

**Pull request merged**